--- nRF5_SDK_17.1.0_ddde560/components/softdevice/common/nrf_sdh_freertos.c	2023-09-26 16:47:21.528956490 +0200
+++ nRF5_SDK_17.1.0_ddde560/components/softdevice/common/nrf_sdh_freertos_new.c	2023-10-02 20:20:46.561220184 +0200
@@ -107,3 +107,14 @@
         APP_ERROR_HANDLER(NRF_ERROR_NO_MEM);
     }
 }
+
+
+void nrf_sdh_freertos_task_pause() 
+{
+    vTaskSuspend(m_softdevice_task);
+}
+
+void nrf_sdh_freertos_task_resume() 
+{
+    vTaskResume(m_softdevice_task);
+}
