diff --git a/include/board_pico_config.h b/include/board_pico_config.h
index d013d4c..9d3b341 100644
--- a/include/board_pico_config.h
+++ b/include/board_pico_config.h
@@ -31,16 +31,16 @@
 
 // PIO config
 #define PROBE_SM 0
-#define PROBE_PIN_OFFSET 2
+#define PROBE_PIN_OFFSET 3
 #define PROBE_PIN_SWCLK (PROBE_PIN_OFFSET + 0) // 2
 #define PROBE_PIN_SWDIO (PROBE_PIN_OFFSET + 1) // 3
 // Target reset config
-#define PROBE_PIN_RESET 0
+#define PROBE_PIN_RESET 2
 
 // UART config
-#define PICOPROBE_UART_TX 4
-#define PICOPROBE_UART_RX 5
-#define PICOPROBE_UART_INTERFACE uart1
+#define PICOPROBE_UART_TX 0
+#define PICOPROBE_UART_RX 1
+#define PICOPROBE_UART_INTERFACE uart0
 #define PICOPROBE_UART_BAUDRATE 115200
 
 #define PICOPROBE_USB_CONNECTED_LED 25
